sdk/storage/azure-storage-blob/azure/storage/blob/_shared/policies.py

```diff
@@ -542,6 +542,8 @@ def send(self, request):
                         continue
                 break
             except AzureError as err:
+                if isinstance(err, AzureSigningError):
+                    raise err
```

Comment:
Will need to push this change to all packages since its shared code, only need to test in one though.